src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -401,6 +401,28 @@ The resulting sitemap looks like this:
 ...
 ```
 
+### xslURL
+
+This sets the URL of an XSL stylesheet that enables you to style and prettify your sitemap. An example of a XSL stylesheets for sitemaps can be found [here](https://github.com/pedroborges/xml-sitemap-stylesheet).
+
+:::note
+The URL can be either relative or absolute. If it is relative, it will be resolved relative to the `site` URL.
+:::
```

Comment:
```suggestion
The URL of an XSL stylesheet to style and prettify your sitemap. 

The value set can be either a path relative to your configured `site` URL for a local stylesheet, or can be an absolute URL link to an external stylesheet.
```
This linked to a package with instructions that wasn't the same usage as what was described here, so I think just pointing people there isn't as helpful as providing JUST a URL that's an example they can drop in and test with.

So I think we'll just remove that for now, but I'd be open to (even a follow up PR) adding a good link to a stylesheet people can point to directly (not a package): is there one people can just drop in and try immediately? Is there some known directory people can browse somewhere? Is this a common thing people showcase anywhere?

So, doesn't need to block THIS PR, but would be happy to have an addition to it that helps people discover existing stylesheets, or learn how to write their own!